web/app.js

```diff
@@ -868,14 +869,17 @@ const PDFViewerApplication = {
       PDFJSDev.test("MOZCENTRAL || GENERIC")
     ) {
       // Only trigger the fallback once so we don't spam the user with messages
-      // for one PDF.
-      if (this.fellback) {
+      // for one PDF. It should only be triggered once the user has clicked
+      // within the PDF.
+      if (!PDFViewerApplication._hasInteracted) {
+        if (!this._featureId) {
+          this._featureId = featureId;
+        }
         return;
       }
-      this.fellback = true;
```

Comment:
Unless I'm mistaken, with these changes you no longer prevent the fallback bar from being triggered *multiple* times (so the comment above no longer holds).

It's probably possible to come up with something better implementation wise than what I'm suggesting below, but it seems that you may still need the `this.fellback` boolean (or similar) and that something like the following is closer to what you probably want here (at least from a functionality point-of-view):
```javascript
if (!this._featureId) { <-- I'm assuming that you want to still capture the featureId value from the *first* invocation of this method.
  this._featureId = featureId;
}
if (!this._hasInteracted) {
  return;
}
if (this.fellback) {
  return;
}
this.fellback = true;
this.externalServices.fallback(
  // The rest of the method here...
```